test/parallel_api/ranges/std_ranges_test.h

```diff
@@ -0,0 +1,362 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include <oneapi/dpl/execution>
+
+#include "support/test_config.h"
+
+#include "support/utils.h"
+
+#define _ENABLE_STD_RANGES_TESTING (_ONEDPL___cplusplus >= 202002L)
```

Comment:
Should we define this macro near the definition of 'TEST_DPCPP_BACKEND_PRESENT` ?